com.bardsoftware.constable/server/src/main/kotlin/com/bardsoftware/constable/server/ConstableServer.kt

```diff
@@ -0,0 +1,41 @@
+package com.bardsoftware.constable.server
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.validate
+import com.github.ajalt.clikt.parameters.types.int
+import java.sql.DriverManager
+import java.sql.SQLException
+import java.util.logging.Level
+import java.util.logging.Logger
+
+private val logger: Logger = Logger.getLogger(ConstableServer::class.java.name)
+
+class ConstableServer : CliktCommand() {
+    private val url by option(help = "url").default("127.0.0.1")
```

Comment:
I suggest using relatively standard names of postgres connection options which consist of "pg" prefix and host/user/database/port/password suffix: `--pg-host`, `--pg-database`, etc.

With these names the semantics of options are kinda unclear. What might mean `./constable-server --url 10.240.0.10 --name foo` ? Run server on host 10.240.0.10 with the name foo?